aws/eip_test.go

```diff
@@ -0,0 +1,84 @@
+package aws
+
+import (
+	"testing"
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/gruntwork-io/cloud-nuke/util"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/stretchr/testify/assert"
+)
+
+func createTestEIPAddress(t *testing.T, session *session.Session, name string) ec2.Address {
+	svc := ec2.New(session)
+	result, err := svc.AllocateAddress(&ec2.AllocateAddressInput{})
+	if err != nil {
+		assert.Fail(t, errors.WithStackTrace(err).Error())
+	}
+
+	output, err := svc.DescribeAddresses(&ec2.DescribeAddressesInput{
+		AllocationIds: []*string{result.AllocationId},
+	})
+
+	if err != nil {
+		assert.Fail(t, errors.WithStackTrace(err).Error())
+	}
+
+	return *output.Addresses[0]
+}
+
+func TestListEIPAddress(t *testing.T) {
```

Comment:
Do you have a test that checks if the tagging actually works? E.g., a test that creates an EIP, waits 2 min, creates another, and sets `excludeAfter` to 1 min, and makes sure only one of the EIPs is returned? You also want to check that tagging works properly, so if that test waits another min, both should be returned.